react/src/Attachment/AttachmentDropzone.tsx

```diff
@@ -16,7 +16,7 @@ export const AttachmentDropzone = ({
   isDragActive,
 }: AttachmentDropzoneProps): JSX.Element => {
   const styles = useAttachmentStyles()
-
+  const filePlural = inputProps.multiple ? 'files' : 'file'
```

Comment:
nit: since already setting some string, might as well set the entire link text here (and update the constant name to be more readable). Also `useMemo` for consistency

```suggestion
  const chooseFileText = useMemo(
    () => `Choose file${inputProps.multiple ? 's' : ''}`,
    [inputProps.multiple],
  )
```